salt/reactors/edx/pulsar_mitx.sls

```diff
@@ -0,0 +1,8 @@
+pulsar_mitx:
+  local.slack.post_message:
+    - tgt: {{ data['id'] }}
+    - kwarg:
+       channel: "#devops"
+       message: "Hubblestack Pulsar FMI - Detected change - Host:`{{ data['id'] }}` - File modified:`{{ data['path'] }}` - User:`{{ data['stats']['user'] }}`"
```

Comment:
It might be useful to also post the entirety of the data that is sent from hubblestack so that we don't lose that information.